Mage.Sets/src/mage/cards/u/UnfinishedBusiness.java

```diff
@@ -0,0 +1,125 @@
+package mage.cards.u;
+
+import java.util.UUID;
+
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.*;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.filter.StaticFilters;
+import mage.filter.predicate.Predicates;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCardInYourGraveyard;
+
+/**
+ *
+ * @author Codermann63
+ */
+public final class UnfinishedBusiness extends CardImpl {
+
+    private static final FilterCard auraOrEquipmentFilter = new FilterCard("Aura or Equipment card");
+
+    static {
+        auraOrEquipmentFilter.add(Predicates.or(
+                SubType.EQUIPMENT.getPredicate(),
+                SubType.AURA.getPredicate()
+        ));
+    }
+
+    public UnfinishedBusiness(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{3}{W}{W}");
+        this.getSpellAbility().addTarget(new TargetCardInYourGraveyard(1,1, StaticFilters.FILTER_CARD_CREATURE));
+        this.getSpellAbility().addTarget(new TargetCardInYourGraveyard(0, 2, auraOrEquipmentFilter));
+        // Return target creature card from your graveyard to the battlefield,
+        // then return up to two target Aura and/or Equipment cards from your graveyard to the battlefield attached to that creature.
+        this.getSpellAbility().addEffect(new UnfinishedBusinessEffect());
+    }
+
+    private UnfinishedBusiness(final UnfinishedBusiness card) {
+        super(card);
+    }
+
+    @Override
+    public UnfinishedBusiness copy() {
+        return new UnfinishedBusiness(this);
+    }
+}
+
+class UnfinishedBusinessEffect extends OneShotEffect{
+
+    UnfinishedBusinessEffect() {
+        super(Outcome.PutCreatureInPlay);
+        staticText = "Return target creature card from your graveyard to the battlefield, then return up to two target Aura and/or Equipment cards from your graveyard to the battlefield attached to that creature. <i>(If the Auras can not enchant that creature, they remain in your graveyard.)</i>";
+    }
+
+    private UnfinishedBusinessEffect(final UnfinishedBusinessEffect effect) {super(effect);}
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer((source.getControllerId()));
+        if (controller == null){
+            return false;
+        }
+
+        // Return target creature from the graveyard to the battlefield
+        Card targetCreature = game.getCard(source.getTargets().getFirstTarget());
+
+        if (targetCreature != null){
+            controller.moveCards(targetCreature, Zone.BATTLEFIELD, source, game);
+        }
```

Comment:
We might want a `game.getState().processAction(game)` in here, to guard against edge cases where continuous effects determine whether the Aura can be attached to the creature.

Very contrived example: return Grim Strider while you have four cards in hand (making it a 2/2), Runner's Bane should attach to it.